privaterelay/templates/faq.html

```diff
@@ -6,65 +6,58 @@
-        </h2>
-        <p class="faq-answer">
-          Some sites may not accept an email address that includes a subdomain (ie, the “relay” portion of @relay.firefox.com) and others have stopped accepting all addresses except those from Gmail, Hotmail, or Yahoo accounts. As Firefox Relay grows in popularity and issues more aliases, our service might be placed on a blocklist. If you are not able to use a Relay alias, please let us know.
-        </p>
-      </section>
-      <section class="faq">
-        <h2 class="faq-headline">
-          Can I make up my own relay address using the @relay.firefox.com domain?
-        </h2>
-        <p class="faq-answer">
-           Not currently, but we are considering new features including letting you create your own addresses with a designated domain.
-        </p>
-      </section>
-      <section class="faq">
-        <h2 class="faq-headline">
-          What happens if Mozilla shuts down the Firefox Relay service?
-        </h2>
-        <p class="faq-answer">
-          We will provide users with advance notice that they will need to change the email address of any of their accounts that are using relay addresses.
-        </p>
-      </section>
+      <div class="faq-col flx flx-col">
+        <section class="faq">
+          <h2 class="faq-headline">How can I join the private beta?</h2>
+          <p class="faq-answer">
+            First, you’ll need a Firefox account. Then, use your account to sign up for the waiting list. When a spot opens, we’ll send you an email with a link you can use to complete registration and start using Relay.
+          </p>
+        </section>
+        <section class="faq">
+          <h2 class="faq-headline">What about spam?</h2>
+          <p class="faq-answer">
+            While Relay does not filter for spam, our email partner Amazon SES does block spam and malware. If Relay forwards messages you don’t want, you can update your Relay settings to block messages from the alias forwarding them.
+          </p>
+          <p class="faq-answer">
+            If you see a broader problem of unwanted email from all of your aliases, please report this to us so we can consider adjusting the SES spam thresholds for this service.
+          </p>
+          <p class="faq-answer">
+            If you report these as spam, your email provider will see Relay as the source of spam, not the original sender.
+          </p>
+        </section>
+        <section class="faq">
+          <h2 class="faq-headline">Why won’t a site accept my Relay alias?</h2>
+          <p class="faq-answer">
+            Some sites may not accept an email address that includes a subdomain (ie, the “relay” portion of @relay.firefox.com) and others have stopped accepting all addresses except those from Gmail, Hotmail, or Yahoo accounts. As Firefox Relay grows in popularity and issues more aliases, our service might be placed on a blocklist. If you are not able to use a Relay alias, please let us know.
```

Comment:
Similar: can we link "please let us know" to https://addons.mozilla.org/en-US/firefox/addon/private-relay/ ?